asab/web/rest/json.py

```
@@ -304,8 +304,11 @@ async def validator(*args, **kwargs):
 				kwargs['json_data'] = data
 			except fastjsonschema.exceptions.JsonSchemaException as e:
 				raise aiohttp.web.HTTPBadRequest(reason=str(e))
-			except Exception as e:
-				Lex.error("JSON validation error. Reason: {}".format(e), struct_data={"path": request.path, "method": request.method})
+			except Exception:
```

Ruff (0.15.17)

[warning] Do not catch blind exception: `Exception`

(BLE001)